examples/Extension/AcmeExtension.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
```

Comment:
either ensure that we never include the PHP tag or that the configuration option is respected in the new directive.